cyences_app_for_splunk/default/macros.conf

```diff
@@ -935,6 +935,28 @@ iseval = 0
 [cs_email_increase_over_baseline_limit]
 definition = 10
 
+[cs_email_suspicious_attachment_extension]
+definition = file_extension_type IN ("pl", "py", "rb", "sh", "bat", "exe", "dll", "cpl", "com", "js", "vbs", "ps1", "reg","swf", "cmd", "go")
+
+[cs_email_suspicious_subjects_and_title(1)]
+args = fieldname
+definition = $fieldname$ IN ("*dhl*", "*ups*", "*delivery*", "*parcel*", "*label*", "*invoice*", "*postal*", "*fedex*", "*usps*", "*express*", "*shipment*", "*Banking/Tax*","*shipment*", "*new order*")
+
+[cs_email_known_abuse_web_service_link]
+definition = link_domain IN ("*pastebin.com*", "*discord*", "*telegram*","t.me")
```

Comment:
Shall we use lookup for this instead of macro for this list? What are your views?